core/audits/metrics/max-potential-fid.js

```diff
@@ -92,6 +108,7 @@ class MaxPotentialFID extends Audit {
       type: 'debugdata',
```

Comment:
We won't see the `observedLoafs` if they were all below FCP in this case. The function will still return undefined because there is no max duration/blocking events after FCP.

Reply:
Thanks! Good catch